A33-Fault-Injection.md

```diff
@@ -106,6 +106,10 @@ Note that for both `X-envoy-fault-abort-request-percentage` and `x-envoy-fault-d
 
 > ...the percentage of request to apply aborts to and must be greater or equal to 0 and its maximum value is capped by the value of the numerator of percentage field.
 
+Note that if any header is invalid, for example, passing non-numerical value as percentage header, the invalid headers will be ignored.
+
+Note if there are multiple valid headers with the same key name, gRPC doesn't make no promise about which one will be picked at the end.
```

Comment:
The phrasing here is very hard to understand.  Suggest changing "gRPC doesn't make no promise about which one will be picked at the end" to "gRPC implementations may choose any one of them".

Reply:
Thanks. This is cleaner.